src/components/HypercertImage.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from "react";
+import { PlasmicHypercertImage, DefaultHypercertImageProps } from "./plasmic/hypercerts_core/PlasmicHypercertImage";
+import { HTMLElementRefOf } from "@plasmicapp/react-web";
+import { HypercertMetadata } from "../types/metadata";
+
+export interface HypercertImageProps extends DefaultHypercertImageProps {
+  metadata: HypercertMetadata
+  rootRef?: React.Ref<HTMLDivElement>;
+  projectLogo?: React.ReactNode;
+  collectionLogo?: React.ReactNode;
+  percentOwnership?: number;
+}
+
+function HypercertImage_(props: HypercertImageProps, ref: HTMLElementRefOf<"div">) {
```

Comment:
Instead of Plasmic it might be cleaner to parameterize this component with default values so that you basically get the Hypercert SVG that we've designed, with the option for SDK users to tweak it as they desire by -for instance- injecting a custom color scheme